src/adapters/adapter.js

```diff
@@ -5,6 +5,24 @@ class Adapter {
     this.store = store;
   }
 
+  _setCacheTime(storeName, cacheKey){
+    if(!this.store.get(storeName)){
+        this.store.set(storeName, {});
+    }
+    const cachedRepos = this.store.get(storeName);
+    cachedRepos[cacheKey] = new Date().getTime();
+    this.store.set(storeName, cachedRepos);
+  }
+
+  _reloadCache(storeName, cacheKey, cb, reloadAfterDay = 29){
+    const cachedRepos = this.store.get(storeName);
+    const cachedRepoTime = cachedRepos[cacheKey];
+    const timeDiff = (cachedRepoTime - new Date().getTime()) / (1000 * 60 * 60 * 24);
```

Comment:
Looks like a magic number here, I think we should constant.

It's not clear to me. Invalidating the cache after 24 hours seems not a good approach. Do we have any other ways to invalidate the cache? The code in repo may change during that period, how can we manage that change?

Reply:
To be clear: I did not set it to 24 hours. It's the `reloadAfterDay` value: 30 days.

What you concern is already addressed by Buu in his comment: https://github.com/ovity/octotree/issues/729#issuecomment-495483665

What do you think about this approach? To me, I really like his idea.